test/acceptance/behave/requirements.txt

```diff
@@ -6,4 +6,4 @@ requests==2.7.0
 xmltodict==0.9.2
 fabric==1.10.2
 pymongo==3.0.3
-git+https://visitor:visitor@pdihub.hi.inet/fiware/fiware-iotqaUtils@develop
\ No newline at end of file
+git+https://github.com/telefonicaid/iot-qa-tools.git
```

Comment:
I don't like it too much, as experience tells that depending on "moving pictures" is a continous source of pain, cry and havoc :) However, I understand that, al least until NGSIv2 get implemented and the QA libreary continues "growing" with new methods, this is the only solution.

I understand that `pip install --upgrade` as to be re-run each time there are changes in `https://github.com/telefonicaid/iot-qa-tools.git` that needs to be updated?